stagemonitor-core/src/main/java/org/stagemonitor/core/elasticsearch/ElasticsearchClient.java

```diff
@@ -64,7 +64,8 @@ public ElasticsearchClient(final CorePlugin corePlugin, final HttpClient httpCli
 			PooledResourceMetricsRegisterer.registerPooledResource(pooledResource, Stagemonitor.getMetric2Registry());
 		}
 		this.httpClient = httpClient;
-		if (esAvailabilityCheckIntervalSec > 0) {
```

Comment:
I'm not sure I follow, do you just want the isOnlyLogElasticsearchMetricReports call removed? There still needs to be an ElasticseachClient instance as many places expect it to exist.